drivers/hmis/app/models/hmis/hud/household.rb

```diff
@@ -48,6 +48,10 @@ class Hmis::Hud::Household < Hmis::Hud::Base
     where(any_wip: false)
   end
 
+  scope :all_exited, -> do
+    where(all_exited: true)
+  end
```

Comment:
I think this is the same as
```
  scope :all_exited, -> do
    where.not(latest_exit: nil)
  end
```

Right? If so I think that's probably better than adding more complexity to the view. Exit dates cannot be in the future.

Reply:
I don't think they're the same but maybe I'm mistaken. If we have a 3-person household where one person has been exited and the other two are still active, then wouldn't `latest_exit` be set to the date the one person was exited? If so, wouldn't the above be satisfied if any person is exited? I think the request here was to make it so that this would only be satisfied if all members of the household are exited.

Looking at the view, it looks like the code for `latest_exit` is:

```sql
CASE WHEN BOOL_OR("Exit"."ExitDate" IS NULL) THEN
    NULL
  ELSE
    MAX("Exit"."ExitDate")
  END AS latest_exit,
```

That should translate to "if no enrollments in this household have an exit date, then set to `null`; Otherwise set to the most recent exit date". I don't think that the presence of `last_exited` then would indicate that ALL members are exited, only that ANY member is exited.

Or am I missing something here?

Reply:
Ah, yeah, that's correct. I thought that `latest_exit` indicated the most recent exit regardless of whether or not other household members were active. Looking at the view code again I see that `latest_exit` should only be set if all members are exited. I'll adjust it.